File: src/test/java/models/layout/LayoutTests.java
package models.layout;

import org.junit.jupiter.api.Nested;
import org.junit.jupiter.api.Tag;
import org.junit.jupiter.api.Test;

import static org.assertj.core.api.AssertionsForClassTypes.assertThat;

public class LayoutTests
{
    @Nested
    @Tag("Unit")
    class UnitTests
    {
        @Test
        public void should_haveListOfRollingsStock()
        {
            //assign

            final Layout layout = new Layout();

            //act

            //assert
            assertThat(layout).isNotNull();
        }
    }
}

File: src/test/java/dev/paigewatson/layoutmaster/models/layout/LayoutTests.kt
package dev.paigewatson.layoutmaster.models.layout

import org.assertj.core.api.AssertionsForClassTypes
import org.junit.jupiter.api.Nested
import org.junit.jupiter.api.Tag
import org.junit.jupiter.api.Test

class LayoutTests {
    @Nested
    @Tag("Unit")
    internal inner class UnitTests {
        @Test
        fun should_haveListOfRollingsStock() {
            //assign
            val layout = Layout()

            //act

            //assert
            AssertionsForClassTypes.assertThat(layout).isNotNull
        }
    }
}
